### PR TITLE
Bug fix/active failover fix windows 3.4

### PR DESCRIPTION
**Pull request by KVS85**
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/2750/
http://jenkins01.arangodb.biz:8080/view/3.4/job/arangodb-3.4-windows-matrix/168/

### CODE REVIEW

File: tests/js/client/active-failover/readonly.js [hunk 1]
```diff
@@ -352,15 +368,15 @@ function ActiveFailoverSuite() {
...
+    //testReadFromFollower: function () {
+    //X-Arango-Allow-Dirty-Read: true
+    //},
 
     testLeaderAfterFailover: function () {
-
       assertTrue(checkInSync(currentLead, servers));
       assertEqual(checkData(currentLead), 10000);
-
+
```
Comment (dothebart) [inline tests/js/client/active-failover/readonly.js]: pls remove whitespace change

File: tests/js/client/active-failover/readonly.js [hunk 2]
```diff
@@ -373,6 +389,7 @@ function ActiveFailoverSuite() {
       let oldLead = currentLead;
       // await failover and check that follower get in sync
       currentLead = checkForFailover(currentLead);
+      return;
```
Comment (dothebart) [inline tests/js/client/active-failover/readonly.js]: You're disabling a lot of code here, is this intentional?

File: js/client/modules/@arangodb/testsuites/resilience.js [hunk 3]
```diff
@@ -93,6 +93,7 @@ function activeFailover (options) {
   let testCases = tu.scanTestPaths(testPaths.active_failover);
   options.activefailover = true;
   options.singles = 4;
+  options.disableMonitor = true;
```
Comment (graetzer) [inline js/client/modules/@arangodb/testsuites/resilience.js]: what is this option doing ?

Comment (KVS85) [reply js/client/modules/@arangodb/testsuites/resilience.js]: That disables procdump.

**Comment by KVS85** [comment]
- http://jenkins01.arangodb.biz:8080/view/3.4/job/arangodb-3.4-windows-matrix/170/
- http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/2751/

**Pull request merged**